apps/ledger-live-mobile/docs/llm_e2e_testing.md

```diff
@@ -0,0 +1,200 @@
+# Ledger Live Mobile end-to-end testing
+
+This project uses [Detox](https://github.com/wix/Detox) and [Jest](https://jestjs.io/) for end-to-end testing the LLM application. Detox is a mobile end-to-end testing tool developed by Wix, and is specifically built for React Native applications. Please refer to the documentation of those projects for the specifics. In this readme you will find the key setup and workflow steps for testing a flow in Ledger Live Mobile.
+
+---
+---
+
+## Full script to install iOS and Android debug versions from scratch
+
+> Ensure you have Android Studio and Xcode installed with the relevant development/emulator tools installed (see 'Local Environment Setup' below).
+
+```bash
+pnpm clean && 
```

Comment:
remove all `&&`, it's better for documentation to have simple one line instructions